### ISSUE TITLE
Fail: sh ds build --install

### ISSUE THREAD

**Issue by aaronschachter** [DoSomethingArchive/legacy-website#145]
Running from /vagrant:

```
Wiping build directory...
Running drush make...
Beginning to build /vagrant/build-dosomething.make.                  [ok]
drupal-7.23 downloaded.                                              [ok]
dosomething cloned from git@github.com:DoSomething/paraneue.git.     [ok]
Linking modules
ln: failed to create symbolic link `/vagrant/html/profiles/dosomething/modules/dosomething': No such file or directory
Linking themes
ln: failed to create symbolic link `/vagrant/html/profiles/dosomething/themes/dosomething': No such file or directory
Installing site...
You are about to DROP all tables in your 'dosomething' database. Do you want to continue? (y/n): y
Starting Drupal installation. This takes a few seconds ...           [ok]
WD php: Exception: Sorry, the profile you have chosen cannot be      [error]
loaded. in install_load_profile() (line 1348 of
/vagrant/html/includes/install.core.inc).
WD php: Warning: Cannot modify header information - headers already  [warning]
sent by (output started at
/usr/share/php/drush/includes/output.inc:38) in drupal_send_headers()
(line 1216 of /vagrant/html/includes/bootstrap.inc).
Exception: Sorry, the profile you have chosen cannot be loaded. in install_load_profile() (line 1348 of /vagrant/html/includes/install.core.inc).
Drush command terminated abnormally due to an unrecoverable error.   [error]
Granting basic permissions...
Command role-add-perm needs a higher bootstrap level to run - you    [error]
will need to invoke drush from a more functional Drupal environment
to run this command.
The drush command 'rap anonymous user access content' could not be   [error]
executed.
Drush was not able to start (bootstrap) the Drupal database.         [error]
Hint: This may occur when Drush is trying to:
 * bootstrap a site that has not been installed or does not have a
configured database. In this case you can select another site with a
working database setup by specifying the URI to use with the --uri
parameter on the command line. See `drush topic docs-aliases` for
details.
 * connect the database through a socket. The socket file may be
wrong or the php-cli may have no access to it in a jailed shell. See
http://drupal.org/node/1428638 for details.

Drush was attempting to connect to: 
 Drupal version         :  7.23                  
 Site URI               :  http://vagrant        
 Default theme          :  garland               
 Administration theme   :  garland               
 PHP executable         :  /usr/bin/php          
 PHP configuration      :  /etc/php5/cli/php.ini 
 PHP OS                 :  Linux                 
 Drush version          :  6.1.0                 
 Drush configuration    :                        
 Drush alias files      :                        
 Drupal root            :  /vagrant/html         
 Site path              :  sites/default         
 File directory path    :  sites/default/files   


Command role-add-perm needs a higher bootstrap level to run - you    [error]
will need to invoke drush from a more functional Drupal environment
to run this command.
The drush command 'rap authenticated user access content' could not  [error]
be executed.
Drush was not able to start (bootstrap) the Drupal database.         [error]
Hint: This may occur when Drush is trying to:
 * bootstrap a site that has not been installed or does not have a
configured database. In this case you can select another site with a
working database setup by specifying the URI to use with the --uri
parameter on the command line. See `drush topic docs-aliases` for
details.
 * connect the database through a socket. The socket file may be
wrong or the php-cli may have no access to it in a jailed shell. See
http://drupal.org/node/1428638 for details.

Drush was attempting to connect to: 
 Drupal version         :  7.23                  
 Site URI               :  http://vagrant        
 Default theme          :  garland               
 Administration theme   :  garland               
 PHP executable         :  /usr/bin/php          
 PHP configuration      :  /etc/php5/cli/php.ini 
 PHP OS                 :  Linux                 
 Drush version          :  6.1.0                 
 Drush configuration    :                        
 Drush alias files      :                        
 Drupal root            :  /vagrant/html         
 Site path              :  sites/default         
 File directory path    :  sites/default/files   


Replacing settings.php file...
Updating database...
<h1>Additional uncaught exception thrown while handling exception.</h1><h2>Original</h2><p>PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table &amp;#039;dosomething.system&amp;#039; doesn&amp;#039;t exist: SELECT name, schema_version FROM {system} WHERE type = :type; Array
(
    [:type] =&amp;gt; module
)
 in drupal_get_installed_schema_version() (line 155 of /vagrant/html/includes/install.inc).</p><h2>Additional</h2><p>PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table &amp;#039;dosomething.system&amp;#039; doesn&amp;#039;t exist: SELECT * FROM {system} WHERE type = &amp;#039;theme&amp;#039; OR (type = &amp;#039;module&amp;#039; AND status = 1) ORDER BY weight ASC, name ASC; Array
(
)
 in system_list() (line 165 of /vagrant/html/includes/module.inc).</p><hr />Drush command terminated abnormally due to an unrecoverable error.   [error]
Clearing caches...
<h1>Additional uncaught exception thrown while handling exception.</h1><h2>Original</h2><p>PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table &amp;#039;dosomething.blocked_ips&amp;#039; doesn&amp;#039;t exist: SELECT 1 FROM {blocked_ips} WHERE ip = :ip; Array
(
    [:ip] =&amp;gt; 127.0.0.1
)
 in drupal_is_denied() (line 1915 of /vagrant/html/includes/bootstrap.inc).</p><h2>Additional</h2><p>PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table &amp;#039;dosomething.system&amp;#039; doesn&amp;#039;t exist: SELECT * FROM {system} WHERE type = &amp;#039;theme&amp;#039; OR (type = &amp;#039;module&amp;#039; AND status = 1) ORDER BY weight ASC, name ASC; Array
(
)
 in system_list() (line 165 of /vagrant/html/includes/module.inc).</p><hr />Drush command terminated abnormally due to an unrecoverable error.   [error]
Build complete.
vagrant@precise64:/vagrant$ 

```

**Comment by mchitten** [comment]
Can't confirm this.  It just worked for me.  Probably obvious questions, but...
1. You're running within the vagrant shell?
2. You're running from /vagrant?

If both of those are true, try without the `sh` part e.g. `ds build --install`.

**Comment by desmondmorris** [comment]
Yah, that error will happen if you are outside of the vagrant shell

**Comment by aaronschachter** [comment]
I've been in the shell... (meaning just within the same terminal that vagrant ssh is runnign correct?)

Have you guys updated your build-dosomething.make with the latest dev version?

Running `drush make build-dosomething.make html` manually doesn't create the site properly:

```
vagrant@precise64:/vagrant$ drush make build-dosomething.make html
Beginning to build build-dosomething.make.                           [ok]
drupal-7.23 downloaded.                                              [ok]
dosomething cloned from git@github.com:DoSomething/paraneue.git.     [ok]
vagrant@precise64:/vagrant$ 
```

Notice no modules were installed, and inspecting the html directory, there's no dosomething profile.

Commenting out the paraneue stuff and running the build does run as expected:

```
vagrant@precise64:/vagrant$ drush make build-dosomething.make html
Beginning to build build-dosomething.make.                           [ok]
drupal-7.23 downloaded.                                              [ok]
dosomething cloned from git@github.com:DoSomething/dosomething.git.  [ok]
Found makefile: drupal-org.make                                      [ok]
Project admin_menu contains 3 modules: admin_devel, admin_menu_toolbar, admin_menu.
```

etc

**Comment by aaronschachter** [comment]
This was resolved by #133
